opencti-platform/opencti-graphql/src/utils/emailTemplates/footer.ts

```diff
@@ -0,0 +1,39 @@
+export const FOOTER_TEMPLATE = `
+                        <tr>
+                           <td>
+                              <table id="footer" border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" bgcolor="#f5f8fa" style="width: 100%; min-width: 100%;">
+                                 <tbody>
+                                    <tr>
+                                       <td align="center" valign="top">
+                                          <table border="0" cellpadding="0" cellspacing="0" height="100%" width="500" style="width: 500px;">
+                                             <tbody>
+                                                <tr>
+                                                   <td align="center" valign="top">
+                                                      <table cellpadding="0" border="0" cellspacing="0" width="100%" style="color: #425b76; background-color: ; font-size: 14px; width: 100%; margin: initial; min-width: 100%; line-height: 24px">
+                                                         <tbody>
+                                                            <tr>
+                                                               <td align="center" valign="middle" style="padding: 5px 0 65px;">
+                                                                  <p style="font-size: 12px; color: #516f90">Copyright &copy; 2024 OpenCTI<br>Powered by <a style="color: #001bda; text-decoration:none;" href="https://filigran.io" target="_blank">Filigran</a></p>
+                                                               </td>
+                                                            </tr>
+                                                         </tbody>
+                                                      </table>
+                                                   </td>
+                                                </tr>
+                                             </tbody>
+                                          </table>
+                                       </td>
+                                    </tr>
+                                 </tbody>
+                              </table>
+                           </td>
+                        </tr>
+                     </tbody>
+                  </table>
+               </td>
+            </tr>
+         </tbody>
+      </table>
+   </body>
+</html>
```

Comment:
body and html and the 2 above tag are not opened here, so better to remove from footer IMO